lib/urlwatch/filters.py

```diff
@@ -419,9 +477,15 @@ def _get_filtered_elements(self):
         if root is None:
             return []
         if self.filter_kind == 'css':
-            return root.cssselect(self.expression)
+            selected_elems = root.cssselect(self.expression)
+            excluded_elems = root.cssselect(self.exclude) if self.exclude else None
         elif self.filter_kind == 'xpath':
-            return root.xpath(self.expression)
+            selected_elems = root.xpath(self.expression)
+            excluded_elems = root.xpath(self.exclude) if self.exclude else None
+        if excluded_elems is not None:
```

Comment:
If `filter_kind` is not `'css'` and not `'xpath'`, `excluded_elems` will never be set? At least introduce a `else:` branch before and set `excluded_elems = None` or set before the line `if self.filter_kind == 'css':`. Thanks :)